bin/cliain/src/commands.rs

```diff
@@ -108,9 +124,9 @@ pub enum Command {
 
     /// Change the validator set for the session after the next
     ChangeValidators {
-        /// The new validators
-        #[clap(long, value_delimiter = ',')]
-        validators: Vec<String>,
+        /// The new reserved validators list
+        #[clap(long)]
+        change_validators_args: ChangeValidatorArgs,
```

Comment:
This means we have to paste json into cmd line right? Could you also add a support for passing path to file with a json?

Reply:
Done:
```bash
marcin@marcin-Latitude-3520 ~/g/a/b/cliain (non-reserved-nodes-support-cliain) [1]> cat test.json                                                                                                   (base) 
{
  "reserved_validators": [
    "5D34dL5prEUaGNQtPPZ3yN5Y6BnkfXunKXXz6fo7ZJbLwRRH",
    "5GBNeWRhZc2jXu7D55rBimKYDk8PGk8itRYFTPfC8RJLKG5o"
  ],
  "non_reserved_validators": [
    "5Dfis6XL8J2P6JHUnUtArnFWndn62SydeP8ee8sG2ky9nfm9",
    "5F4H97f7nQovyrbiq4ZetaaviNwThSVcFobcA5aGab6167dK"
  ],
  "committee_size": {
    "reserved_seats": 2,
    "non_reserved_seats": 2
  }
}
```
And now it is possible just to do `./target/release/cliain change-validators --change-validators-args test.json` :)